hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -514,7 +514,7 @@ private FileStatusAdapter toFileStatusAdapter(OzoneFileStatus status,
         keyInfo.getModificationTime(),
         keyInfo.getModificationTime(),
         status.isDirectory() ? (short) 00777 : (short) 00666,
-        owner,
+        StringUtils.defaultIfEmpty(keyInfo.getOwnerName(), owner),
         owner,
```

Comment:
`group` should also have the same treatment?

```suggestion
        StringUtils.defaultIfEmpty(keyInfo.getOwnerName(), owner),
        StringUtils.defaultIfEmpty(keyInfo.getOwnerName(), owner),
```

Reply:
The current change only involves `owner`, and `owner` and `group` are two separate attributes, so I think we're better off leaving `group` unchanged.